yarn-error.log

```diff
@@ -0,0 +1,8572 @@
+Arguments: 
```

Comment:
lol, can you please add `yarn-error.log` to .gitignore while you are here?

Reply:
Ugh. This bloody yarn-error.log today 😡

Reply:
You don't want to put it in the gitignore?